test/controllers/SubscriptionControllerSpec.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import play.api.test.FakeRequest
+import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
+
+class SubscriptionControllerSpec extends UnitSpec with WithFakeApplication {
+
+  "Calling .subscribe" should {
```

Comment:
Can you change this just to be a little more specific. Maybe the following would be better:

`"Calling the .subscribe method of the SubscriptionController" should {`